### PR TITLE
Added sceAppUtilAppEventParseLiveArea prototype.

### PR DESCRIPTION
**Pull request by Rinnegatamante**

### CODE REVIEW

File: include/psp2/apputil.h [hunk 1]
```diff
@@ -158,6 +158,9 @@ int sceAppUtilShutdown();
 //! Receive app event
 int sceAppUtilReceiveAppEvent(SceAppUtilAppEventParam *eventParam);
 
+//! Parse received app event from LiveArea
+int sceAppUtilAppEventParseLiveArea(SceAppUtilAppEventParam *eventParam, char *buffer);
```
Comment (DaveeFTW) [inline include/psp2/apputil.h]: Is const-ness correct here?

Comment (devnoname120) [reply include/psp2/apputil.h]: [This](https://github.com/Rinnegatamante/vitaQuake/blob/82af45c89559e48491791c1368855f17e146ec57/source/sys_psp2.c#L502) seems to indicate it's not `const`.

Comment (DaveeFTW) [reply include/psp2/apputil.h]: for SceAppUtilAppEventParam

**Comment by Rinnegatamante** [comment]
Fixed and rebased

**Pull request merged**